packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentMethods.css

```diff
@@ -15,3 +15,7 @@
     font-weight: 600;
     justify-self: start;
 }
+
+.no_payment_method {
+    padding: 1.5rem;
```

Comment:
To get this component more aligned with the others (shipping address/method) lets just add `padding: 2rem;` to the `root` class above, and remove this class. Also, remove the padding from the `payment_method` class.